### DISCUSSION TITLE
unlimited ban with bantime.multipliers

### DISCUSSION THREAD

**Discussion by novazur972** [fail2ban/fail2ban#3824] [General]
Hi,

If I use bantime.multipliers = 1 5 -1, will the IP be banned forever at its third ban please ?
If no, how to use bantime.multipliers (or may be bantime.formula) to have an unlimited ban after X bans ?

Thanks

**Comment by sebres** [comment]
No, it would not, `multipliers` is a part of formula and would generate negative seconds, see:
https://github.com/fail2ban/fail2ban/blob/3361fb0805630ff8eabd4531ee1e3bd97cee6ce6/fail2ban/server/jail.py#L251-L253

It is not possible to force persistent ban with `multipliers`, but...

Why someone needs it at all? 
The bantime of something like 5 weeks is fully enough (and in worst case this means ca. 10 attempts per year).
I would recommend not larger than something like `5w`, so `bantime.maxtime = 5w`. 
What will be fully enough to ban recidive evildoers (repeating the attempt latter) as well as not to waste the system resources by large fail2ban and net-filter tables and chains (consider the lot of dynamic IPs, etc).

In paranoidal mode just use something like `1y`. But something larger than a year doesn't really make sense at all.
